util.js

```diff
@@ -1,11 +1,14 @@
+const path = require("path");
 const got = require("got");
 
 const getDependenciesWithTypes = async (packageName, version) => {
   if (!packageName) {
-    throw new Error("Package name missing");
+    console.warn("\nPackage name missing, looking for local package.json");
```

Comment:
I think we can remove this `if` statement altogether. What do you think?

Reply:
Sounds great to me! I'll push the change